src/components/Labelled/Labelled.tsx

```diff
@@ -48,6 +57,17 @@ export default function Labelled({
     </div>
   ) : null;
 
+  const characterCountMarkup = showCharacterCount &&
+    maxLength && (
```

Comment:
Would there ever be a case where this might be used without a maxLength? Does it make sense to only render it if the `maxLength` is provided?

Reply:
I had the same feeling. If there is no `maxLength`, I would change the copy to `n characters used` or simply `n characters`.